lib/wab/impl/shell.rb

```diff
@@ -30,8 +30,10 @@ def initialize(config)
         @logger.level = config[:verbosity] unless @logger.nil?
         @http_dir     = (config['http.dir'] || File.join(base, 'pages')).gsub('$BASE', base)
         @http_port    = (config['http.port'] || 6363).to_i
+        @export_proxy = config['export_proxy']
+        @export_proxy = true if @export_proxy.nil? # The default is true if not present.
```

Comment:
This can be shortened to
```ruby
@export_proxy = config['export_proxy'] || true
```

Reply:
Not really as the first part could return false which is a valid value.